proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -301,6 +314,27 @@ message Params {
   uint32 historical_entries = 4 [(gogoproto.moretags) = "yaml:\"historical_entries\""];
   // bond_denom defines the bondable coin denomination.
   string bond_denom = 5 [(gogoproto.moretags) = "yaml:\"bond_denom\""];
+  // validator_bond_factor is required as a safety check for tokenizing shares and
+  // delegations from liquid staking providers
+  string validator_bond_factor = 6 [
```

Comment:
@sampocs 
field with index 6 is already busy for `min_commission_rate` in SDK 0.46+:

https://github.com/cosmos/cosmos-sdk/blob/38ea2823ad233c862142f643b9a63c5ebd3c5051/proto/cosmos/staking/v1beta1/staking.proto#L301-L306

```
  // min_commission_rate is the chain-wide minimum commission rate that a validator can charge their delegators
  string min_commission_rate = 6 [
    (gogoproto.moretags)   = "yaml:\"min_commission_rate\"",
    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
    (gogoproto.nullable)   = false
  ];
```

We need to reserve it there and have LSM-related fields starting from idx 7.

Reply:
good call! done [here](url)